### DISCUSSION TITLE
Osi

### DISCUSSION THREAD

**Discussion by ApdalaElaan** [community/community#34001] [Other Feature Feedback, Questions, & Ideas]
I am very sorry that my facebook account that I have been using for 12 years is locked due to the authorization code.

**Comment by WendelGustavo** [comment]
### **Goodnight!** 
If you haven't tried yet, send an email to support, I already suffered from invalidation of my account and after a brief conversation with support, my case was resolved and the account was reactive. Possibly they will be able to give you all the support you need.
_Hope this helps._
